src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,43 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numers = new int[12];
+    private static final int minusOne = -1;

```

Comment:
Irrelevant (redundant in information) name.
Constants should be written UPPERCASE (e.g., MINUS_ONE)